RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -716,13 +716,70 @@ void finalizeMaximize(boolean speculative)
 			addToMaximize("+equip " + toEquip);
 		}
 	}
+
+	if(in_wereprof() && auto_haveDarts()) //Absolutely need darts for Professor. Should level up darts while Werewolf too
+	{
+		if(is_werewolf())
+		{
+			addBonusToMaximize($item[Everfull Dart Holster], 1000);
+		}
+		else
+		{
+			addToMaximize("+equip " + $item[Everfull Dart Holster]);
+		}
+	}
+
+	if(in_wereprof() && !is_werewolf() && (possessEquipment($item[biphasic molecular oculus]) || possessEquipment($item[triphasic molecular oculus]))) //Want that Advanced Research as a professor
+	{
+		//exclude certain locations as professor that require specific outfits (the War, the Goblin King)
+		if(!($locations[The Battlefield (Frat Uniform), The Battlefield (Hippy Uniform), Frat House, Hippy Camp, Frat House (Frat Disguise), Hippy Camp (Hippy Disguise), Next to that barrel with something burning in it,
+		Out by that rusted-out car, over where the old tires are, near an abandoned refrigerator, Sonofa Beach, The Themthar Hills, McMillicancuddy's Barn, McMillicancuddy's Pond, McMillicancuddy's Back 40,
+		McMillicancuddy's Other Back 40, Cobb\'s Knob Barracks, Cobb\'s Knob Harem, Throne Room] contains my_location()))
+		{
+			if(possessEquipment($item[biphasic molecular oculus]))
+			{
+				addToMaximize("+equip " + $item[biphasic molecular oculus]);
+			}
+			else
+			{
+				addToMaximize("+equip " + $item[triphasic molecular oculus]);
+			}
+		}
+	}
+
+	if(in_wereprof() && !is_werewolf() && (possessEquipment($item[high-tension exoskeleton]) || possessEquipment($item[ultra-high-tension exoskeleton]) || possessEquipment($item[irresponsible-tension exoskeleton]))) //Want that damage avoidance
+	{
+		//exclude certain locations as professor that require specific outfits (the War, the Goblin King)
+		if(!($locations[The Battlefield (Frat Uniform), The Battlefield (Hippy Uniform), Frat House, Hippy Camp, Frat House (Frat Disguise), Hippy Camp (Hippy Disguise), Next to that barrel with something burning in it,
+		Out by that rusted-out car, over where the old tires are, near an abandoned refrigerator, Sonofa Beach, The Themthar Hills, McMillicancuddy's Barn, McMillicancuddy's Pond, McMillicancuddy's Back 40,
+		McMillicancuddy's Other Back 40, Cobb\'s Knob Barracks, Cobb\'s Knob Harem, Throne Room] contains my_location()))
+		{
+			if(possessEquipment($item[high-tension exoskeleton]))
+			{
+				addToMaximize("+equip " + $item[high-tension exoskeleton]);
+			}
+			else if(possessEquipment($item[ultra-high-tension exoskeleton]))
+			{
+				addToMaximize("+equip " + $item[ultra-high-tension exoskeleton]);
+			}
+			else
+			{
+				addToMaximize("+equip " + $item[irresponsible-tension exoskeleton]);
+			}
+		}
+	}
+
 	
 	if(auto_haveSpringShoes())
 	{
 		if(item_amount($item[ultra-soft ferns])<4 || item_amount($item[crunchy brush])<4) // collect the spring shoes potions
 		{
 			addBonusToMaximize($item[spring shoes], 200);
 		}
+		else if(in_wereprof() && my_hp() < 0.5*my_maxhp() && my_hp() > 0)
```

Comment:
would this be useful for any class? Wondering if the `in_wereprof()` should be removed

Reply:
It probably would be useful. I figured that most other classes would use restores they could buy as opposed to Werewolf where you can't buy anything so I will remove it since it is handy.